evm/interchainTokenFactory.js

```diff
@@ -237,6 +279,7 @@ if (require.main === module) {
                 'deployRemoteInterchainToken',
                 'registerCanonicalInterchainToken',
                 'deployRemoteCanonicalInterchainToken',
+                'registerAllGatewayTokens',
```

Comment:
Make sure to add these variables as options below: `assetApi, tokenInfoApi, batchSize`